### ISSUE TITLE
dRonine flags

### ISSUE THREAD

**Issue by ResinSmoker** [d-ronin/dRonin#2219]
Everytime i attempt to download or install dRonin it flags with Google (Chrome), Malwarebytes and Avast. I've attempted several times to contact the vendors to submit an exception but after several months still no luck. Would it be possible to contact the vendors on your end to see if they'll hear your voice?

![dronine blocked by malwarebytes](https://user-images.githubusercontent.com/10391485/42268454-dd6715f2-7f7b-11e8-8339-fa3a66f81aff.png)

For the meantime, I've added exclusions for both Malwarebytes and Avast.

**Comment by glowtape** [comment]
I wonder if the RunGuard stuff and its shared memory antics is tripping AV scanners.

**Comment by tracernz** [comment]
The other thing I think it could be is the breakpad handler hooking all the signals.

**Comment by tracernz** [comment]
Interesting chrome comments on that: https://codereview.chromium.org/1416133003/patch/1160001/1170033

-e- They embed the handler app in chrome.exe and the decision to run it, or the normal chrome exe comes from argv https://codereview.chromium.org/1416133003/patch/1160001/1170017

**Comment by glowtape** [comment]
Hmmm that'd be annoying.

It's the drgcs.exe resulting that trips everything. I'm gonna look into it tomorrow, uploading a stock build to VirusTotal.com and see if it does the same (or whether I need a Jenkins build), and then replace the sharedmem stuff with a global mutex as barrier on Windows and see if it still happens. Should exclude that possibility.

**Comment by glowtape** [comment]
Must be some weird ass codegen stuff. A debug build of drgcs.exe of next triggers only one AV out of 67 over on VirusTotal.com, called Cylance, which is supposed to be some AI shit. A release build triggers 15 of them, which is an improvement over Wired, which triggers a whopping 35. Replacing the shared mem stuff with a global mutex still triggers 15 of them.

**Comment by glowtape** [comment]
Ain't the crash reporting nor the Runguard stuff in general. Excluding them still triggers stuff. Dafuq?

**Comment by tracernz** [comment]
Maybe they really are just trolling us.

**Comment by mlyle** [comment]
gonna call this duplicate of #2221
